pkg/odo/cli/cli.go

```diff
@@ -175,6 +176,8 @@ func NewCmdOdo(name, fullName string) *cobra.Command {
 		debug.NewCmdDebug(debug.RecommendedCommandName, util.GetFullName(fullName, debug.RecommendedCommandName)),
 		pipelines.NewCmdComponent(pipelines.RecommendedCommandName, util.GetFullName(fullName, pipelines.RecommendedCommandName)),
 		pipelines.NewCmdInit(pipelines.InitRecommendedCommandName, util.GetFullName(fullName, pipelines.InitRecommendedCommandName)),
+		manifest.NewCmdComponent(manifest.RecommendedCommandName, util.GetFullName(fullName, manifest.RecommendedCommandName)),
+		manifest.NewCmdInit(manifest.InitRecommendedCommandName, util.GetFullName(fullName, manifest.InitRecommendedCommandName)),
```

Comment:
Rename method `manifest.NewCmdComponent()` to `manifest.NewCmdManifest()` and remove this line:
`manifest.NewCmdInit(manifest.InitRecommendedCommandName, util.GetFullName(fullName, manifest.InitRecommendedCommandName)),`.   

So these two lines become this.  (just one line below)
``` 		manifest.NewCmdManifest(manifest.RecommendedCommandName, util.GetFullName(fullName, manifest.RecommendedCommandName)),```

Reply:
Oh, definitely, this has bugged me for ages, thanks :-)